### ISSUE TITLE
Adservice can't be built on windows

### ISSUE THREAD

**Issue by simonz130** [GoogleCloudPlatform/microservices-demo#128]
When doing 
```
skaffold run --default-repo=gcr.io/REDACTED_PROJECT_ID
```
the build fails with this:
```
/bin/sh: 1: ./gradlew: Permission denied
FATA[1951] build failed: building [gcr.io/opencensus-demos/adservice]: build artifact: running build: The command '/bin/sh -c ./gradlew downloadRepos' returned a non-zero code: 126
```
Running build instruction is not Windows OS compatible.

**Comment by simonz130** [comment]
Update: I was able to find the RC for this:
1. It has to do with line endings difference between windows and linux.
2. There are issues with docker for Windows that require this command twice:
```
RUN chmod +x ./gradlew
```
Once before **downloadRepos** script and second time before **installDist**.
It's somewhat similar issue to the one from here: https://github.com/gliderlabs/logspout/issues/238

I will be sending PR with fixes soon.

**Comment by ahmetb** [comment]
Hmm, actually, I wrote the code for Windows that adds files into a docker build context back in 2015. https://github.com/moby/moby/issues/11047 Back then, we used to add all files with executable bits (+x) because they were getting lost.

Unless something has changed (found an issue here https://github.com/moby/moby/issues/30511) that should still work. What do you do when you go inside the build step container and stat the file?

**Comment by ahmetb** [comment]
Fixed in #146.
